site/specs/numbers.json

```
@@ -3324,11 +3325,11 @@
               "type": "string",
               "default": null
             },
-            "example": null
+            "example": "1234567"
           },
           {
             "name": "tn",
-            "description": "<p><b class=\"required\">*</b> A Telephone Number (TN) that is referenced in the order</p>\n",
+            "description": "<b class=\"required\">*</b> A Telephone Number (TN) that is referenced in the order",
             "in": "query",
             "required": false,
```

```suggestion
            "description": "A Telephone Number (TN) that is referenced in the order",
            "in": "query",
            "required": true,
```
I believe you'd like to mark this field as required by the original description.